src/main/java/com/google/devtools/build/lib/rules/genrule/GenRuleBase.java

```diff
@@ -115,28 +149,35 @@ public ConfiguredTarget create(RuleContext ruleContext)
       return null;
     }
 
-    String baseCommand = ruleContext.attributes().get("cmd", Type.STRING);
+    CommandType cmdType = cmdTypeAndAttr.first;
+    String cmdAttr = cmdTypeAndAttr.second;
+    boolean expandToWindowsPath = cmdType != CommandType.BASH;
```

Comment:
Hm, doesn't PowerShell also understand Unix-style paths?
(And PowerShell runs on Linux too!)

Reply:
That's a good point! If Powershell supports Unix-style path well enough, I think we should just use path with forward slash. I'll change this and see if tests still pass.

Reply:
SGTM!